tests/test_resources/test_envs/test_env.py

```diff
@@ -343,25 +37,168 @@ def np_summer(a, b):
     return int(np.sum([a, b]))
 
 
-@pytest.mark.clustertest
-def test_conda_call_fn(cluster):
-    conda_dict = _get_conda_env(name="c")
-    conda_env = rh.env(conda_env=conda_dict, reqs=["pytest", "numpy"])
-    fn = rh.function(np_summer).to(system=cluster, env=conda_env)
-    result = fn(1, 4)
-    assert result == 5
-
+class TestEnv(TestResource):
 
-@unittest.skip("Map does not work properly following Module refactor.")
-@pytest.mark.clustertest
-def test_conda_map_fn(cluster):
-    conda_dict = _get_conda_env(name="test-map-fn")
-    conda_env = rh.env(conda_env=conda_dict, reqs=["pytest", "numpy"])
-    map_fn = rh.function(np_summer, system=cluster, env=conda_env)
-    inputs = list(zip(range(5), range(4, 9)))
-    results = map_fn.starmap(inputs)
-    assert results == [4, 6, 8, 10, 12]
+    MAP_FIXTURES = {"resource": "env"}
```

Comment:
@dongreenberg this is still running all the base resource tests for the resource fixtures in addition to the env fixtures. how should that be resolved?